etcd3/client.py

```diff
@@ -24,7 +24,7 @@
 }
 
 
-def _translate_exeption(exc):
+def _translate_exception(exc):
```

Comment:
This is a good change but is has nothing to do in /this/ patch.
Can you do another PR? it should be quickly mergeable.